app/gui/src/project-view/components/visualizations/TableVisualization.vue

```diff
@@ -197,7 +197,11 @@ function formatText(params: ICellRendererParams) {
     .replaceAll('>', '&gt;')
 
   if (textFormatterSelected.value === 'off') {
-    return htmlEscaped.replace(/^\s+|\s+$/g, '&nbsp;')
+    const replaceLinks = htmlEscaped.replace(
+      /https?:\/\/([a-zA-Z0-9-]+(\.[a-zA-Z0-9-]+)+)(:[0-9]+)?(\/[-()_.!~*';/?:%@&=+$,A-Za-z0-9]*)?/,
```

Comment:
This regexp could use some test cases. `test.each` may be helpful. I would test what the RE matches rather than the expected result of inserting the `<a>` tag so that it is independent of the details of link rendering.

Reply:
Actually, we have a `LINKABLE_URL_REGEX` that might be appropriate here